AmericanMedicalAssociation/AMA-style-guide
# EWL-4151 Add all the favicons.

## Ticket(s)

Please do not submit a Pull Request without a relevant ticket! If you are creating a new pattern or feature, create an issue describing the need for this feature Github **first** and then link to it below.

**Github Issue**

NA

**Jira Ticket**

- [EWL-4151: Change drupal icon to AMA icon in browser tab](https://issues.ama-assn.org/browse/EWL-4151)


## Description

Add favicons to the codebase. This duplicates the code and assets from the corp site.

## To Test

- `gulp serve`
- open a raw page, like http://localhost:3000/patterns/00-atoms-00-intro-00-intro/00-atoms-00-intro-00-intro.html?1509636730032
- observe all the favicon code, check that it does fun things across browsers.


## Relevant Screenshots/GIFs

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks

Remaining tasks?


## Additional Notes

Anything more to add?
